mage/query-modules/cpp/degree_centrality.md

```diff
@@ -0,0 +1,148 @@
+---
+id: degree-centrality
+title: degree_centrality
+sidebar_label: degree_centrality
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+{children}
+</span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-degree_centrality-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/cpp/degree_centrality_module/algorithm/degree_centrality_module.cpp)
+
+## Abstract
+
+**Degree Centrality** is the basic measurement of centrality that refers to the
+number of edges adjacent to a node. For directed graphs, we define an in-degree
+measure, which is defined as the number of in-coming edges, and an out-degree
+measure, defined as the number of out-going edges.
+
+Let $A = (a_{i,j})$ be the adjacency matrix of a directed graph. The in-degree centrality $x_{i}$ of node $i$ is given by: $$x_{i} = \sum_k a_{k,i}$$ or in matrix form ($1$ is a vector with all components equal to unity): $$x = 1 A$$ The out-degree centrality $y_{i}$ of node $i$ is given by: $$y_{i} = \sum_k a_{i,k}$$ or in matrix form: $$y = A 1$$
```

Comment:
Please check if /$1/$ is rendering well on your system... bcs it isn't on mine, but it could be something going on with me locally

Reply:
doesn't show on github either, and locally I have some problem displaying .md files right now. It works on Overleaf though. I removed the dollar signs

Reply:
Kate said there are some issues with this extension now... so i don't know what to do here... any ideas? Image?